### PR TITLE
prepare a 0.1.3 release

### LINKED ISSUE #217: atomicapp 0.1.2 labels are broken
Looks like the labeling with 0.1.2 is not working. When I `install` I get no artifacts and when I `run` I get a broken container:

```
[root@nule fooa]$ atomic info fooapache
io.projectatomic.nulecule.specversion: 0.0.2
io.projectatomic.nulecule.providers: = kubernetes,docker
io.projectatomic.nulecule.atomicappversion: 0.1.2       RUN  docker run -it --rm  --privileged -v /home/dustymabe:/atomicapp -v /run:/run -v /:/host --net=host --name  -e NAME= -e IMAGE=  -v  run  /atomicapp       STOP docker run -it --rm  --privileged -v /home/dustymabe:/atomicapp -v /run:/run -v /:/host --net=host --name  -e NAME= -e IMAGE=  -v  stop  /atomicapp       INSTALL docker run -it --rm  --privileged -v /home/dustymabe:/atomicapp -v /run:/run  --name  -e NAME= -e IMAGE=  -v  install  --destination /atomicapp /application-entity
[root@nule fooa]$ atomic install fooapache
[root@nule fooa]$ ls -a
.  ..
[root@nule fooa]$ atomic run fooapache
/usr/bin/docker create -t -i --name fooapache fooapache /bin/sh
fooapache
[root@nule fooa]$ docker ps -a
CONTAINER ID        IMAGE               COMMAND                CREATED             STATUS                     PORTS               NAMES
f17d6c6c87e4        fooapache           "/usr/bin/atomicapp    12 seconds ago      Exited (2) 6 seconds ago                       fooapache
```

### PR DESCRIPTION
**Pull request by goern**
This fixes #217 and supersedes and closed #218

### CODE REVIEW

**Comment by goern** [comment]
please also merge https://github.com/projectatomic/nulecule/pull/149 after release of 0.1.3

**Comment by sub-mod** [comment]
needed for https://github.com/cockpit-project/cockpit/pull/2572

**Comment by sub-mod** [comment]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -2,11 +2,10 @@ FROM centos:centos7
 
 MAINTAINER Vaclav Pavlin <vpavlin@redhat.com>
 
-# it's a good practice to have all labels in one layer
-LABEL io.projectatomic.nulecule.atomicappversion 0.1.2 \
-      RUN  "docker run -it --rm \${OPT1} --privileged -v `pwd`:/atomicapp -v /run:/run -v /:/host --net=host --name \${NAME} -e NAME=\${NAME} -e IMAGE=\${IMAGE} \${IMAGE} -v \${OPT2} run \${OPT3} /atomicapp" \
-      STOP "docker run -it --rm \${OPT1} --privileged -v `pwd`:/atomicapp -v /run:/run -v /:/host --net=host --name \${NAME} -e NAME=\${NAME} -e IMAGE=\${IMAGE} \${IMAGE} -v \${OPT2} stop \${OPT3} /atomicapp" \
-      INSTALL "docker run -it --rm \${OPT1} --privileged -v `pwd`:/atomicapp -v /run:/run  --name \${NAME} -e NAME=\${NAME} -e IMAGE=\${IMAGE} \${IMAGE} -v \${OPT2} install \${OPT3} --destination /atomicapp /application-entity"
+LABEL io.projectatomic.nulecule.atomicappversion="0.1.3" \
+      RUN="docker run -it --rm \${OPT1} --privileged -v `pwd`:/atomicapp -v /run:/run -v /:/host --net=host --name \${NAME} -e NAME=\${NAME} -e IMAGE=\${IMAGE} \${IMAGE} -v \${OPT2} run \${OPT3} /atomicapp" \
```
Comment (aweiteka) [inline Dockerfile]: I'm still seeing these in the `\${NAME}` format. These need to be in the form of `--name NAME`.

**Comment by stefwalter** [comment]
FYI, we will be merging https://github.com/cockpit-project/cockpit/pull/2572 which requires a minimum atomicapp version of 0.1.3

**Comment by aweiteka** [comment]
LGTM.

**Pull request merged**